addon/webextension/background/selectorLoader.js

```diff
@@ -0,0 +1,83 @@
+/* globals browser, catcher */
+window.selectorLoader = (function () {
+  const exports = {};
+  // These modules are loaded in order, and some need to be listed before others
+  // due to dependencies:
+  const scripts = [
+    "catcher.js",
+    "background/selectorLoader.js",
+    "selector/callBackground.js",
+    "clipboard.js",
+    "makeUuid.js",
+    "build/shot.js",
+    "randomString.js",
+    "domainFromUrl.js",
+    "build/inlineSelectionCss.js",
+    "selector/documentMetadata.js",
+    "selector/util.js",
+    "selector/ui.js",
+    "selector/shooter.js",
+    "selector/uicontrol.js"
+  ];
+
+  exports.unloadIfLoaded = function () {
+    return browser.tabs.executeScript({
+      code: "window.selectorLoader && window.selectorLoader.unloadModules()"
+    }).then(result => {
+      return result && result.toString() === "true";
```

Comment:
It was the simplest thing I could get to work. I couldn't figure out what `result` actually was. `typeof result` is `object` and `result == true` is `false`. I notice `console.log(result)` was the string `"true"` so I went with that. I rather do something less weird too.